libs/my-stack/feature-stacks/src/lib/overview/stack.component.ts

```diff
@@ -0,0 +1,17 @@
+import { CommonModule } from "@angular/common";
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import { MatCardModule } from "@angular/material/card";
+import { Stack } from "@my-stack/shared/api-my-stack";
+
+@Component({
+  selector: "my-stack-stack",
```

Comment:
Maybe we should consider a shorter prefix such as my / mys / myst. It works for material (mat), and would solve this issue :D

Reply:
I've created an issue regarding this topic, see #38